README.md

```diff
@@ -21,7 +21,7 @@ Pa11y Webservice provides scheduled accessibility reports for multiple URLs. It
 Setup
 -----
 
-Pa11y Webservice requires [Node.js][node] 4+ and [PhantomJS][phantom]. See the [Pa11y documentation][pa11y-docs] for more information on these dependencies. Pa11y Webservice also requires [MongoDB][mongo] to be installed and running.
+Pa11y Webservice requires [Node.js][node] 4+ and [PhantomJS][phantom]. See the [Pa11y documentation][pa11y-docs] for more information on these dependencies. Pa11y Webservice also requires [MongoDB][mongo] to be installed and running.  Instructions for installing Mongo are available on their website, but for simple use-cases, installing Mongo using your OS package manager such as `brew install mongodb` or `apt-get install mongodb` is likely to be sufficient.
```

Comment:
We could do with having this on the Pa11y Dashboard docs too, actually - I'll create an issue. 

I'd simplify this, too, though, and add explicit instructions for starting up the service and where to find the docs. Maybe something like:

>Pa11y Webservice also requires [MongoDB][mongo] to be installed and running. Install with a package manager, e.g. `brew install mongodb` (for Homebrew),  or `apt-get install mongodb` (for Debian), then start up the service with `mongod`. See the [MongoDB install guide](https://docs.mongodb.org/manual/installation/) for more information.